### PR TITLE
Rm27 si ci

### PR DESCRIPTION
**Pull request by remymartinDev**

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -4240,6 +4240,21 @@ though, and has since been slightly modified)-->
...
+                    </token>
+                    <token>si</token>
+                </marker>
+            </pattern>
+            <message>La conjonction 'si' induit une condition.</message>
+            <suggestion>fois-ci</suggestion>
+            <example correction="fois-ci">Malheureusement, encore une énième rechute, qui je crois cette <marker>fois si</marker> sonnera le glas de la fin de carrière de Jesse</example>
+            <example>Mais cette fois, si tu y penses</example>
+        </rule>
         <rule id="POLE_2" name="pôle" default="temp_off">
```
Comment (jaumeortola) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:4258]: The exception for the comma (`<exception>,</exception>`) is useless.
Possible FP: "Mais cette fois si tu y penses". In this case, we can suggest also the missing comma.

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 2]
```diff
@@ -4240,6 +4240,21 @@ though, and has since been slightly modified)-->
         </rule>
     </category>
     <category id="CAT_HOMONYMES_PARONYMES" name="Confusion d’homonymes et paronymes">
+        <rule id="SI_CI" name="cette fois si" default="temp_off">
+            <pattern>
+                <token>cette</token>
+                <marker>
+                    <token>fois
+                        <exception>,</exception>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:4248]: `scope="next"`?

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 3]
```diff
@@ -4240,6 +4240,21 @@ though, and has since been slightly modified)-->
...
+                    </token>
+                    <token>si</token>
+                </marker>
+            </pattern>
+            <message>La conjonction 'si' induit une condition.</message>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:4253]: "introduit"?

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml (hunk 1 above)
Comment (udomai) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:4258]: > On va y aller tous ensemble cette fois si Paul ne rate pas son train.

If "si" is followed by a subject (subject pronoun or noun), followed by a predicate, this rule will probably give a false positive. You could put this as an AP in this rule and create a second subrule for the missing comma(s).

**Review by udomai — CHANGES_REQUESTED** [review]

**Pull request merged**